health-checker/src/main/java/io/stargate/health/DataStoreHealthChecker.java

```diff
@@ -82,4 +102,45 @@ protected Result check() {
       return Result.unhealthy("Unable to access DataStore: " + e);
     }
   }
+
+  private void ensureTableExists(DataStore dataStore)
```

Comment:
I think that we should have a dedicated table that is used only for this health-check. Other tables (from `system.*`)are mocked by the Stargate.